lib/chanko/test.rb

```diff
@@ -33,7 +33,8 @@ def active_with_activations?(*args)
           active_without_activations?(*args)
         end
       end
-      alias_method_chain :active?, :activations
+      alias_method :active_without_activations?, :active?
+      alias_method :active?, :active_with_activations?
```

Comment:
This replaces `Chanko::Unit::ClassMethods::active?` defined in lib/chanko/unit.rb.

I did not rewrite this with `prepend`  because 1) I am not sure it works with `ClassMethods` module and 2) there may be user code depending on `active_with(out)_activations?` so we need to define them anyway.